### PR TITLE
Add `reset` and `target` options to `combine-events`

### LINKED ISSUE #59: Add  `reset` and `target` options for `combineEvents` method
Seems reasonable for `combineEvents`, as there is a need for `reset` and `target` options

_Originally posted by @doasync in https://github.com/sergeysova/patronum/issues/33#issuecomment-650067147_

### PR DESCRIPTION
**Pull request by doasync**
Closes #59

### CODE REVIEW

File: combine-events/combine-events.test.ts [hunk 1]
```diff
@@ -0,0 +1,796 @@
+/* eslint-disable prettier/prettier */
+import { createEvent, restore, Event } from 'effector';
+import { combineEvents } from './index';
+
+const argumentHistory = (fn) => fn.mock.calls.map(([argument]) => argument);
```
Comment (sergeysova) [inline combine-events/combine-events.test.ts:5]: You can import it from ../test-libary

File: combine-events/index.js [hunk 2]
```diff
@@ -5,19 +5,27 @@ const {
...
+  throw new Error(message);
+};
+
 function combineEvents(argument) {
-  const { events, loc, name = 'unknown' } = readConfig(argument, [
-    'events',
+  const {
+    events,
+    reset,
+    target: givenTarget,
```
Comment (sergeysova) [inline combine-events/index.js:20]:
```suggestion
    target = createEvent({ name: `${name}Target`, loc }),
```

And, please, move `sid` and `loc` and `name` before all other props.

```
    name = ‘unknown’,
```

File: combine-events/combine-events.test.ts (hunk 1 above)
Comment (doasync) [reply combine-events/combine-events.test.ts:5]: The version from `test-library.js` is different:
image

Comment (doasync) [reply combine-events/combine-events.test.ts:5]: I have used the version from effector package. Why not you?

File: combine-events/index.js (hunk 2 above)
Comment (doasync) [reply combine-events/index.js:22]: done

File: combine-events/combine-events.test.ts (hunk 1 above)
Comment (sergeysova) [reply combine-events/combine-events.test.ts:5]: What about to create issue to change our version to version from effector?

Comment (doasync) [reply combine-events/combine-events.test.ts:5]: Sure, but it is not a blocker for this PR

**Comment by sergeysova** [comment]
Is it ready to review?

File: combine-events/combine-events.test.ts (hunk 1 above)
Comment (doasync) [reply combine-events/combine-events.test.ts:5]: #64

**Comment by sergeysova** [comment]
@doasync thank you!

**Pull request merged**